apps/topics-map/Dockerfile

```diff
@@ -5,7 +5,10 @@
 FROM gcr.io/mcback/common:latest
 
 # Install Java
-RUN apt-get -y --no-install-recommends install openjdk-8-jre-headless
+RUN \ 
+    apt-get update && \ 
```

Comment:
fix for https://github.com/mediacloud/backend/pull/771/checks?check_run_id=1937604900#step:13:2904